### PR TITLE
Simplifies the Hello, World tutorial.

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: docs/source/hello-world-state.rst [hunk 1]
```diff
@@ -103,53 +97,50 @@ define an ``IOUState``:
...
 
-  * ``value`` is just a standard int (in Java)/Int (in Kotlin), but ``sender`` and ``recipient`` are of type
+  * ``value`` is just a standard int (in Java)/Int (in Kotlin), but ``lender`` and ``borrower`` are of type
     ``Party``. ``Party`` is a built-in Corda type that represents an entity on the network.
 
-* We've overridden ``participants`` to return a list of the ``sender`` and ``recipient``
-* This means that actions such as changing the state's contract or its notary will require approval from both the
-  ``sender`` and the ``recipient``
+* We've overridden ``participants`` to return a list of the ``lender`` and ``borrower``
+  * Actions such as changing a state's contract or notary will require approval from all the ``participants``
```
Comment (roger3cev) [inline docs/source/hello-world-state.rst]: formatting of the bullets here. When viewed you see an asterisk inline with the above bullet

File: docs/source/hello-world-contract.rst [hunk 2]
```diff
@@ -42,7 +42,7 @@ Just as every Corda state must implement the ``ContractState`` interface, every
             val legalContractReference: SecureHash
         }
 
-A few more Kotlinisms here:
+You can read about function declarations in Kotlin `here <https://kotlinlang.org/docs/reference/functions.html>`_.
 
 * ``fun`` declares a function
```
Comment (roger3cev) [inline docs/source/hello-world-contract.rst]: can we remove these lines around function syntax and just point people to the kotlin docs?

File: docs/source/hello-world-running.rst [hunk 3]
```diff
@@ -134,7 +130,7 @@ We want to create an IOU of 100 with Node B. We start the ``IOUFlow`` by typing:
 
 .. code:: python
 
-    start IOUFlow arg0: 99, arg1: "CN=NodeB,O=NodeB,L=New York,C=US"
+    start IOUFlow arg0: 99, arg1: "NodeB"
```
Comment (roger3cev) [inline docs/source/hello-world-running.rst]: maybe it's clearer to use the argument names instead of arg0/1, etc.

Comment (joeldudleyr3) [inline docs/source/hello-world-running.rst]: I've added both - one for Java and one for Kotlin.

File: docs/source/hello-world-state.rst (hunk 1 above)
Comment (joeldudleyr3) [inline docs/source/hello-world-state.rst]: Done

**Pull request merged**